airbyte-integrations/connectors/source-declarative-manifest/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "5.0.0"
+version = "5.0.1"
```

Comment:
I don't think 5.0.0 has been released so we should be good to keep the same version number

Reply:
edit: it's been released on pypi, but not docker :/